XIVLauncher/App.xaml.cs

```diff
@@ -216,10 +216,32 @@ private void App_OnStartup(object sender, StartupEventArgs e)
             // Check if the accountName parameter is provided, if yes, pass it to MainWindow
             var accountName = string.Empty;
 
-            if (e.Args.Length > 0 && e.Args[0].StartsWith("--account="))
+            if (e.Args.Length > 0)
             {
-                accountName = e.Args[0].Substring(e.Args[0].IndexOf("=", StringComparison.InvariantCulture) + 1);
-                App.Settings.CurrentAccountId = accountName;
+                foreach (string arg in e.Args)
+                {
+                    if (arg.StartsWith("--account="))
+                    {
+                        accountName = arg.Substring(arg.IndexOf("=", StringComparison.InvariantCulture) + 1);
+                        App.Settings.CurrentAccountId = accountName;
+                    }
+                }
+            }
+                
+

```

Comment:
You can combine these for loops to simplify the logic a bit!

Reply:
I've updated the PR to include a simplified loop and dragged in the other two launch args because why not. They'll exit as usual if selected.

I've also tested what happens if a user selects an unsupported language and the enum parser will default to Japanese because it's the first one available. As this will be a command for advanced users, I think this is probably fine since we can make an accompanying FAQ for it if needed.